docs/documentation/ui-rest-api.md

```diff
@@ -172,6 +172,27 @@ Sample response:
 }
 ```
 
+### /api/v1/history/summary (GET)
+
+Returns a list of topology ID submitted by the current user.
```

Comment:
The URL has /history in it but the documentation does not say if it will return all the topologies ever submitted (a true history) or only the current snapshot. If its former we should update the documentation , if its later we should chane the URL to /api/v1/topologIds.